libs/sdk-ui-kit/src/List/InvertableSelect/test/useInvertableSelect.test.ts

```diff
@@ -0,0 +1,248 @@
+// (C) 2022 GoodData Corporation
+
+//import { act } from "@testing-library/react";
+import { renderHook } from "@testing-library/react-hooks";
+import { IUseInvertableSelectProps, selectionStatusType, useInvertableSelect } from "../useInvertableSelect";
+
+const defaultProps: IUseInvertableSelectProps<number> = {
+    items: [],
+    totalItemsCount: 0,
+    getItemKey: (item: number) => item.toString(),
+    searchString: "",
+    isInverted: true,
+    selectedItems: [],
+};
+
+const render = (props: Partial<IUseInvertableSelectProps<number>>) => {
+    const { result } = renderHook(() => useInvertableSelect<number>({ ...defaultProps, ...props }));
+    return result;
+};
+
+describe("useInvertableSelect", () => {
+    describe("selectionState", () => {
+        const Scenarios: Array<
+            [
+                selectionStatusType, //result
```

Comment:
Custom type with lower case first letter is strange, can you change it?
Also, you can put the names in the code, not comments like this, it will then show up in intellisense :)
```ts
[
    result: selectionStatusType,
    desc: string,
    items: number[],
    selectedItems: number[],
    isInverted: boolean,
    totalItemsCount: number,
    searchString: string,
]
```

Reply:
Fixed